### DISCUSSION TITLE
Tinting the whole window instead of just the text area

### DISCUSSION THREAD

**Discussion by Annwan** [kovidgoyal/kitty#5581] [Q&A] [answered]
Is there a way to apply `background_tint` to the whole window instead of just the text area?

With brightly colored backgrounds I find it awkward to have this bright border around the terminal
![Screenshot_20221013_091717](https://user-images.githubusercontent.com/30665559/195528285-840817ca-de11-4dcf-9a43-700d9bcf9adc.png)
The only solution I see right now is pre-tinting the image in a image editor and disabling kitty's tinting, but that means I'd need to redo it every time I decide to change color scheme

**Comment by page-down** [comment, marked as answer]
This issue has been resolved.
Try the nightly release or wait for the next version.

https://github.com/kovidgoyal/kitty/issues/3933

**Reply by Annwan** [reply]
thanks for the swift reply, switching to the nightly version works for me
